Jenkinsfile

```diff
@@ -118,10 +116,6 @@ def buildStep(target, compilerLabel, toolchain, radioShield) {
           }
           execute("mbed new .")
 
-          if ("${radioShield}" == "wifi-ism43362") {
-            execute ("mbed add wifi-ism43362")
-          }
-
```

Comment:
@AnttiKauppila I've seen your commit from yesterday to remove IAR and update targets. But the removal of this is also needed for this PR, because it adds a .lib for for `wifi-ism43362`.

Thanks.

Reply:
Okay, I have permissions and CI passes now